components/HTMLContent.js

```diff
@@ -60,7 +60,12 @@ const HTMLContent = styled(({ content, collapsable, sanitize, ...props }) => {
   let __html = sanitize ? sanitizeHtml(content) : content;
 
   if (collapsable && !isOpen) {
-    __html = getFirstSentenceFromHTML(__html);
+    const firstSentence = getFirstSentenceFromHTML(__html);
+    __html = firstSentence;
+    // Hide "Read full description" if we can display everything in the firstSentence
+    if (firstSentence === __html) {
```

Comment:
@kewitz : Isn't this always gonna be true because of the preceding assignment, `__html = firstSentence;` ? 🤔

Reply:
![](https://media.giphy.com/media/3o85xnoIXebk3xYx4Q/giphy.gif)

Reply:
😸 🤣 🤣